taca/flowcell_parser/classes.py

```diff
@@ -0,0 +1,322 @@
+
+import os
+import csv
+import xml.etree.ElementTree as ET
+import logging
+
+from bs4 import BeautifulSoup #html parser
```

Comment:
Yah!! But I though `bs4` is a standard package of python :sweat_smile: So it isn't ?